scripts/generators/csv_generator.py

```diff
@@ -39,35 +16,29 @@ def generate(ecs_flat: Dict[str, Field], version: str, out_dir: str) -> None:
 def base_first(ecs_flat: Dict[str, Field]) -> List[Field]:
     base_list: List[Field] = []
     sorted_list: List[Field] = []
+
     for field_name in sorted(ecs_flat):
         if '.' in field_name:
             sorted_list.append(ecs_flat[field_name])
         else:
             base_list.append(ecs_flat[field_name])
+
     return base_list + sorted_list
 
 
 def save_csv(file: str, sorted_fields: List[Field], version: str) -> None:
-    open_mode: str = "wb"
-    if sys.version_info >= (3, 0):
-        open_mode: str = "w"
+    open_mode = "w"
```

Comment:
I'd suggest simplifying even more:

`with open(file. "w") as csvfile:`

If we do make this change in this generator, I think it'd make sense to do the same across the repo (e.g. [here](https://github.com/elastic/ecs/blob/main/scripts/generators/es_template.py#L283)).